### PR TITLE
[MCC-399980] Feature/mauth signer example

### PR DESCRIPTION
**Pull request by sbeletskaya-mdsol**
updating documentation for the mauth signer

### CODE REVIEW

File: modules/mauth-signer-apachehttp/README.md [hunk 1]
```diff
@@ -5,37 +5,24 @@ This is an implementation of Medidata Authentication Client Signer to sign the H
...
-            .build();
-
-and then use `httpClient` as normal
-  * Manually signing requests
-
-        final HttpClientRequestSigner httpClientRequestSigner = new HttpClientRequestSigner(configuration);
-        HttpGet request = new HttpGet("http://example.com/resources/1");
-        httpClientRequestSigner.signRequest(request);
+   * MAuth uses [Typesafe Config](https://github.com/typesafehub/config).
+     Create `application.conf` on your classpath with following content.
```
Comment (ChazzMaTazz) [inline modules/mauth-signer-apachehttp/README.md]: "the following content."

File: modules/mauth-signer-apachehttp/src/example/java/MauthRequestInterceptorSignerExample.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            StatusLine status = response.getStatusLine();
+            System.out.println("response code: " + status.getStatusCode()  + " (" + status.getReasonPhrase() + ")");
+            System.out.println("response: " + response.getEntity().getContent().toString());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Example how to use Medidata Authentication Client Signer to sign the Http requests
```
Comment (ChazzMaTazz) [inline modules/mauth-signer-apachehttp/src/example/java/MauthRequestInterceptorSignerExample.java]: Suggest rewriting to: "Example: How to use Medidata authentication client to sign HTTP requests"

File: modules/mauth-signer-akka-http/src/example/resources/application.conf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+app {
+  uuid: ${?APP_MAUTH_UUID}
+  private_key: ${?APP_MAUTH_PRIVATE_KEY}
+}
```
Comment (austek) [inline modules/mauth-signer-akka-http/src/example/resources/application.conf]: please add new line

File: modules/mauth-signer-apachehttp/src/example/resources/application.conf [hunk 4]
```diff
@@ -0,0 +1,4 @@
+app {
+  uuid: ${?APP_MAUTH_UUID}
+  private_key: ${?APP_MAUTH_PRIVATE_KEY}
+}
```
Comment (austek) [inline modules/mauth-signer-apachehttp/src/example/resources/application.conf]: please add new line

File: project/ExampleTesting.scala [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import sbt._
+
+object ExampleTesting {
+  lazy val ExampleTests = config("example") extend (Test)
+  lazy val exampleSettings = Seq(inConfig(ExampleTests)(Defaults.testSettings) : _*)
+}
```
Comment (austek) [inline project/ExampleTesting.scala]: please add new line

**Pull request merged**